ArraysHomework/ArraysHomework.cs

```diff
@@ -66,14 +81,18 @@ public void Arrays()
             {
                 array1[up] = up;
                 array2[down] = down;
-                
+
             }
 
+            int[] combined = new int[50];
+            combined = array1.Concat(array2).ToArray();
```

Comment:
This is probably close how I would approach the problem in real code, but what is it actually doing? How would you solve this using the basic tools of arrays and loops, without Linq methods (methods that need the `using System.Linq;` directive at the top of the file)?